### PR TITLE
Updating ReadMe and apiDefinition for FactSet Connector

### PR DESCRIPTION
**Pull request by KyleVErwin**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by KyleVErwin** [comment]
@microsoft-github-policy-service agree company="FactSet Research Systems"

File: certified-connectors/FactSet/apiDefinition.swagger.json [hunk 1]
```diff
@@ -1,1411 +1,1422 @@
...
+                    }
+                  },
+                  "description": "errors"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Signals - Headlines",
+        "operationId": "get-headlines",
```
Comment (Amjed-Ayoub) [inline certified-connectors/FactSet/apiDefinition.swagger.json:218]: @KyleVErwin all operationIds should be in Camel Case. is it possible to change to "GetHeadlines". 
Please do not change the operationId for the existing operations, only for the new ones. As if you change an existing operationId will break existing flows.

File: certified-connectors/FactSet/apiDefinition.swagger.json [hunk 2]
```diff
@@ -1,1411 +1,1422 @@
...
+          }
+        ]
+      }
+    },
+    "/content/symbology/v3/identifier-resolution": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```
Comment (Amjed-Ayoub) [inline certified-connectors/FactSet/apiDefinition.swagger.json:1268]: @KyleVErwin empty schema is not allowed. please define a proper schema. Please fix all empty schemas in the swagger file.

File: certified-connectors/FactSet/apiDefinition.swagger.json [hunk 3]
```diff
@@ -1,1411 +1,1422 @@
...
+      "tokenUrl": "https://auth.factset.com/as/token.oauth2",
+      "scopes": {}
+    }
+  },
+  "security": [
+    {
+      "oauth2_auth": []
+    }
+  ],
+  "tags": []
```
Comment (Amjed-Ayoub) [inline certified-connectors/FactSet/apiDefinition.swagger.json:1421]: @KyleVErwin 'x-ms-connector-metadata' property is required. Please see out templates [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json#:~:text=%22x%2Dms%2Dconnector,%5D).

File: certified-connectors/FactSet/apiDefinition.swagger.json [hunk 4]
```diff
@@ -1,1411 +1,1422 @@
...
+        },
+        "summary": "Symbology API",
+        "description": "Translate Entity ID and market security symbols into FactSet Permanent Security Identifiers.",
+        "operationId": "get-symbology",
+        "parameters": [
+          {
+            "name": "ids",
+            "in": "query",
+            "required": false,
+            "type": "string"
```
Comment (Amjed-Ayoub) [inline certified-connectors/FactSet/apiDefinition.swagger.json:1279]: @KyleVErwin The 'x-ms-summary' property is required. The 'description' property is required. Please add both to the parameter.

File: certified-connectors/FactSet/apiDefinition.swagger.json [hunk 5]
```diff
@@ -1,1411 +1,1422 @@
...
+          },
+          {
+            "name": "inputSymbolType",
+            "in": "query",
+            "required": false,
+            "type": "string"
```
Comment (Amjed-Ayoub) [inline certified-connectors/FactSet/apiDefinition.swagger.json:1285]: @KyleVErwin The 'x-ms-summary' property is required. The 'description' property is required. Please add both to the parameter.

File: certified-connectors/FactSet/apiDefinition.swagger.json [hunk 6]
```diff
@@ -1,1411 +1,1422 @@
...
+          },
+          {
+            "name": "outputSymbolTypes",
+            "in": "query",
+            "required": false,
+            "type": "string"
```
Comment (Amjed-Ayoub) [inline certified-connectors/FactSet/apiDefinition.swagger.json:1291]: @KyleVErwin The 'x-ms-summary' property is required. The 'description' property is required. Please add both to the parameter.

File: certified-connectors/FactSet/apiDefinition.swagger.json [hunk 7]
```diff
@@ -1,1411 +1,1422 @@
...
+  "paths": {
+    "/book-builder-api/book-list": {},
+    "/book-builder-api/template-list": {},
+    "/book-builder-api/get-book-id/{ticker}": {},
+    "/signals/v2/events/headlines": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
```
Comment (Amjed-Ayoub) [inline certified-connectors/FactSet/apiDefinition.swagger.json:24]: @KyleVErwin The 'default' response should not have schema definition. Schemas should be defined on expected responses only. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses). Please fix all responses.
```
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1signals~1v2~1events~1headlines/get/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1signals~1v2~1events~1details/get/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1signals~1v2~1events~1adaptive-cards/get/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1book-builder-api~1v1~1book-list/get/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1book-builder-api~1v1~1template-list/get/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1book-builder-api~1v1~1create-book-from-template/post/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1book-builder-api~1v1~1download-api-book~1{book_id}/get/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1cognitive~1nlp~1v1~1ner~1entities/post/responses/default

```

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/FactSet/apiDefinition.swagger.json [hunk 8]
```diff
@@ -1260,144 +1387,70 @@
...
-                    "text": {
-                      "type": "string",
-                      "description": "text"
-                    }
-                  },
-                  "description": "data"
-                }
-              }
-            }
+            "type": "string"
```
Comment (Amjed-Ayoub) [inline certified-connectors/FactSet/apiDefinition.swagger.json:1439]: @KyleVErwin 'x-ms-summary' and the 'description' property is required. Please add both to the parameter.

File: certified-connectors/FactSet/apiDefinition.swagger.json [hunk 9]
```diff
@@ -1260,144 +1387,70 @@
...
+          },
+          {
+            "name": "inputSymbolType",
+            "in": "query",
+            "required": false,
+            "type": "string"
```
Comment (Amjed-Ayoub) [inline certified-connectors/FactSet/apiDefinition.swagger.json:1445]: @KyleVErwin 'x-ms-summary' and the 'description' property is required. Please add both to the parameter.

File: certified-connectors/FactSet/apiDefinition.swagger.json [hunk 10]
```diff
@@ -1260,144 +1387,70 @@
...
+          },
+          {
+            "name": "outputSymbolTypes",
+            "in": "query",
+            "required": false,
+            "type": "string"
```
Comment (Amjed-Ayoub) [inline certified-connectors/FactSet/apiDefinition.swagger.json:1451]: @KyleVErwin 'x-ms-summary' and the 'description' property is required. Please add both to the parameter.

File: certified-connectors/FactSet/apiDefinition.swagger.json [hunk 11]
```diff
@@ -1260,144 +1387,70 @@
...
                 }
               }
             }
           }
         },
-        "summary": "NER - Entities",
-        "operationId": "ner-entities",
+        "summary": "Symbology API",
+        "description": "Translate Entity ID and market security symbols into FactSet Permanent Security Identifiers.",
+        "operationId": "get-symbology",
```
Comment (Amjed-Ayoub) [inline certified-connectors/FactSet/apiDefinition.swagger.json:1433]: @KyleVErwin all operationIds should be in Camel Case. Please change to "GetSymbology". Please fix all operationIds in the swagger file.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Review by Amjed-Ayoub — APPROVED** [review]
Hello @KyleVErwin,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**